package/batocera/utils/batocera-wine/batocera-wine

```diff
@@ -5,8 +5,13 @@ set -x
 ACTION=$1
 shift
 
+## Wine detection
+GAMENAME=$1 
+ROMGAMENAME=$(basename "${GAMENAME}")
+WINE_VERSION=$(batocera-settings -e -r core -s windows -g "${ROMGAMENAME}") || WINE_VERSION=$(batocera-settings -e -r core -s windows)
```

Comment:
L11 and L12 are obsolete.
There is always the GAMENAME parsed to the script (afaik!!)

So this will be simply a one liner then
```
WINE_VERSION=$(batocera-settings -e -r core -s windows -g "${ROMGAMENAME}") || WINE_VERSION=proton
```

Reply:
That not true.

Reply:
yes only these